lib/influxer/metrics/metrics.rb

```diff
@@ -162,10 +170,50 @@ def tags
       @attributes.select { |k, _| tag_names.include?(k.to_s) }
     end
 
-    attributes :timestamp
+    def time(parse = false)
+      return unless @time
+
+      return parse_timestamp(@time) if parse
+      @time
+    end
+
+    def time=(val)
+      if val.is_a?(Date) || val.is_a?(Time)
+        @time = parse_time(val)
+      else
+        @time
+      end
+    end
 
     private
 
+    def parse_timestamp(val)
+      case client.time_precision
+      when 'ms'
```

Comment:
Ok.
I will set the default value in config.rb, sure?
Like this:
`time_precision: 'ms',`
Well, this line define default value to 'ms', but allow overwrite this value with configuration file.

To ignore all others time precisions, i need add this code on the `parsed_timestamp`,
`return @timestamp unless client.time_precision == 'ms'`

In this case (using non-standard time precision) the timestamp needs to be converted before assignment. Sure?